### ISSUE TITLE
Permission error in protodune-declad-np04 and now also protodune-declad-np02

### ISSUE THREAD

**Issue by StevenCTimm** [DUNE/data-mgmt-ops#629]
2 days in a row have seen the problem

<img width="1132" alt="Screenshot 2024-06-19 at 6 39 27 PM" src="https://github.com/DUNE/data-mgmt-ops/assets/26069304/206e8c6a-31c7-4b4b-8951-f06bc9f6fd01">

Error shown in log is:

Data copy failed: Run: [ERROR] Server responded with an error: [3010] Unable to open file /eos/experiment/neutplatform/protodune/dune/hd-protodune/04/a6/np04hd_raw_run027289_0004_dataflow0_datawriter_0_20240619T102721.hdf5; Operation not permitted (destination)

In the pictured thing it went from 10:02 to 10:58 in some time zone.

This would be when the DECLAD is trying to copy the file from the dropbox to the final rucio area.
This copy is not done by FTS3 so there's no FTS3 delegation issue.
As before this one resolved itself.

**Comment by StevenCTimm** [comment]
The graph shows 12:02 to 12:58, the logs on screen show 10:02 to 10:58 in UTC.

**Comment by StevenCTimm** [comment]
The proxy is not expired at that time either for the voms proxy life or the base proxy life, so that's not it.
 Also the proxy didn't get changed at the end of the time frame so that also can't be it.

**Comment by StevenCTimm** [comment]
<img width="1282" alt="Screenshot 2024-06-20 at 8 14 34 AM" src="https://github.com/DUNE/data-mgmt-ops/assets/26069304/6efdd04d-4835-462c-8fb1-136244a7c14c">

**Comment by StevenCTimm** [comment]
same problem happened overnight on protodune-declad-np04.

so I have discovered that the EOS tree has different file and directory ownership.
Whenever I log into the protodune-declad-np02 and protodune-declad-np04 (which I did last night) 
and restart the daemon, I leave a kerberos keytab behind, and the resulting xrdcp are done with Kerberos authentication as long as it is valid, and end up being owned by "timm" rather than by np04data.

This does need to be fixed but given the EOS acls user timm can write np04data  owned directories and vice versa it is not a big problem

So we still need to get to the bottom of the Operation Not Permitted error. 

On a secondary level, overnight there were also a lot of timeouts talking to rucio and SAM.
